src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -93,48 +84,105 @@ const RuleDetails = ({ children, rule, intl, topics, header, isDetailsPage }) =>
+                                                />
+                                            </StackItem>
+                                            <StackItem>
+                                                <SeverityLine
+                                                    title={intl.formatMessage(messages.impactLevel, {
+                                                        level: AppConstants.IMPACT_LABEL[rule.impact.impact]
+                                                    })}
+                                                    value={rule.impact.impact}
+                                                    tooltipMessage={intl.formatMessage(messages.impactDescription, {
+                                                        level: AppConstants.IMPACT_LABEL_LOWER[rule.impact.impact]
+                                                    })}
+                                                />
+                                            </StackItem>
+                                        </Stack>
+                                    </Stack>
+                                </SplitItem>
+                            </Split>
                         </StackItem>
                         <hr></hr>
                         <StackItem>{intl.formatMessage(messages.riskofchange)}</StackItem>
-                        <StackItem className='pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm'>
-                            <span className='pf-u-display-inline-flex'>
-                                <Shield
-                                    hasTooltip={false}
-                                    impact={resolutionRisk}
-                                    size={'md'}
-                                    title={AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk] || intl.formatMessage(messages.undefined)} />
-                                <span className={`label pf-u-pl-sm ins-sev-clr-${resolutionRisk}`}>
-                                    {AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk] || intl.formatMessage(messages.undefined)}
-                                </span>
-                            </span>
-                        </StackItem>
-                        <StackItem>
-                            <TextContent>
-                                <Text component={TextVariants.small}>
-                                    {AppConstants.RISK_OF_CHANGE_DESC[resolutionRisk]}
-                                </Text>
-                            </TextContent>
+                        <StackItem className={`pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm`}>
+                            <Split hasGutter>
+                                <SplitItem>
+                                    <InsightsLabel text={AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk]} value={resolutionRisk} hideIcon/>
+                                </SplitItem>
+                                <SplitItem isFilled/>
```

Comment:
again, not sure we need the extra padding

Reply:
same as above